### PR TITLE
Added Ratchet topic

### PR DESCRIPTION
**Pull request by cboden**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

This will provide meta information for the many repositories built using Ratchet when looking at the `ratchet` tag. This should also help find projects built with Ratchet.

### CODE REVIEW

File: topics/ratchet/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+related: php
```
Comment (cheshire137) [inline topics/ratchet/index.md]: You could also include `websockets` and `library`.

File: topics/ratchet/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+created_by: Chris Boden
+display_name: Ratchet
+logo: ratchet.png
+github_url: https://github.com/ratchetphp
+released: May 11, 2012
+short_description: Ratchet is a set of libraries to handle WebSockets asynchronously in PHP.
+topic: ratchet
+url: http://socketo.me/
+---
+Ratchet is a set of libraries to handle WebSockets in PHP. It provides Ratchet for serving, Pawl as a client, and RFC6455 for I/O agnostic protocol handling.
```
Comment (cheshire137) [inline topics/ratchet/index.md]: I think "I/O agnostic" should be hyphenated as a compound adjective.

I'm unfamiliar with Ratchet, so I read a little on http://socketo.me/. The bit about "create real time, bi-directional applications between clients and servers" is interesting, if you wanted to work it into this description.

**Comment by cboden** [comment]
Thanks for the feedback @cheshire137! I've updated the documentation accordingly. 👍

File: topics/ratchet/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+created_by: Chris Boden
+display_name: Ratchet
+logo: ratchet.png
+github_url: https://github.com/ratchetphp
+released: May 11, 2012
+short_description: Ratchet is a set of libraries to handle WebSockets asynchronously in PHP.
+topic: ratchet
+url: http://socketo.me/
+---
+Ratchet is a set of libraries that allows PHP developers to create real time, bi-directional applications between clients and servers over WebSockets. It provides Ratchet for serving, Pawl as a client, and RFC6455 for I/O-agnostic protocol handling.
```
Comment (cheshire137) [inline topics/ratchet/index.md]: I think "real time" should be hyphenated, but "bi-directional" can lose the hyphen and be one word.

It's a little confusing that Ratchet is both the set of libraries but also a single component, if I'm reading it right. "Ratchet is a set of libraries..." versus "It provides Ratchet...". I'm not sure how that could be clarified.

Could "It provides Ratchet" be rephrased to "Ratchet is the server library"?

File: topics/ratchet/index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+related: php, websockets
```
Comment (cheshire137) [inline topics/ratchet/index.md]: How about also "library"? Just thinking if someone very new to programming comes here, https://github.com/topics/library has an explanation of what a library is.

Could also list `client` and `server`.

**Comment by cboden** [comment]
I added more tags and I took a shot at clarifying the long description. How's that @cheshire137?

**Pull request merged**